### ISSUE TITLE
Select element inside iframe not read when navigating with arrows

### ISSUE THREAD

**Issue by RichCaloggero** [nvaccess/nvda#9904]
## Steps to reproduce

If an iframe contains a `select` element, focusing the select and attempting to read it's options with arrow keys causes NVDA to remain silent.

### This select works fine

```
data:text/html,<select aria-label="choose one"><option>item 1</option><option>item 2</option><option>item 3</option></select>
```

### this one does not announce options as you move with arrows

```
data:text/html,<iframe src='data:text/html,<select aria-label="choose one"><option>item 1</option><option>item 2</option><option>item 3</option></select>'></iframe>
```

### Actual behavior:

NVDA does not announce the option text as you move through the select with arrows.

### Expected behavior:

NVDA reads each option's text as you move through the select with arrows.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2019.1.1

#### Windows version:
windows 10

#### Name and version of other software in use when reproducing the issue:
firefox 67.0.4

#### Does the issue still occur after restarting your PC?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

**Comment by XLTechie** [comment]
A few things:

1. Chrome and Edge do not exhibit the behavior described--both combo boxes 
work identically.

2. I can, however, replicate the described behavior in Firefox.

3. There are other circumstances where I have observed this behavior in Firefox, 
not just with iframes, though I can not recall the circumstances off the top of 
my head. It happens to me often however.

4. You can obtain normal functionality by "expanding" the combo box. I.E. by 
pressing space on it. In other words, for the first one while you have to press 
space once to change its options (probably dependent on your mode settings), the 
second one requires that you press space twice, before it will behave normally. 
Also, to leave it, you may have to escape from it twice.

**Comment by Adriani90** [comment]
This works as expected in Firefox 75, Crome and Edge Cromium 81 with NVDA Version: alpha-20015,56a62097. Closing as works for me.
